specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -111,9 +111,9 @@
     "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/availableBalance/default": {
       "get": {
         "tags": [
-          "Credits"
+          "AvailableBalances"
         ],
-        "operationId": "AvailableBalanceByBillingProfile_Get",
+        "operationId": "AvailableBalances_GetByBillingProfile",
```

Comment:
>AvailableBalances_GetByBillingProfile [](start = 24, length = 37)

I suppose this is a breaking change w.r.t SDK..  Customers would have to fix & rebuild their applications to reflect the name change. Are we cool with that ?

Reply:
These are the brand new set of APIs and we are publishing it for the first time.
Docs are not yet out. It would be OK for this time.